opbeat/instrumentation/packages/django/template.py

```diff
@@ -15,3 +15,32 @@ def call(self, module, method, wrapped, instance, args, kwargs):
             name = '<template string>'
         with self.client.capture_trace(name, "template.django"):
             return wrapped(*args, **kwargs)
+
+
+class DjangoTemplateSourceInstrumentation(AbstractInstrumentedModule):
+    name = 'django_template_source'
+    instrument_list = [
+        ('django.template.base', 'Parser.extend_nodelist')
+    ]
+
+    def call(self, module, method, wrapped, instance, args, kwargs):
+        ret = wrapped(*args, **kwargs)
+
+        if len(args) > 0:
+            node = args[1]
```

Comment:
doesn't this explode if `len(args) == 1`? same on line 37 when `len(args) == 2`. Or is it possible that there are less args? `Parser.extend_nodelist` looks like it always takes 3 arguments